boxes/__init__.py

```diff
@@ -1249,15 +1249,30 @@ def regularPolygonHole(self, x, y, r=0.0, d=0.0, n=6, a=0.0, tabs=0):
         if n == 0:
             self.hole(x, y, r=r, tabs=tabs)
             return
+ 
         if r < self.burn:
             r = self.burn + 1E-9
         r_ = r - self.burn
+        
```

Comment:
I am not 100% decide what I think of that. From an API POV the default radius should be a true `0.0` including the over cuts to make things fit. Otoh these holes are mostly not used for constructive purposes. May we can make the default radius `None` and translate that to `burn` but allow to pass `0.0` for getting the over cuts. This still doesn't play well wit making the radius an UI parameter, tough. May be I am overthinking this...

Reply:
@florianfesti It looks like the original author of `regularPolygonalHole` intended this:

`if the radius is less than the burn, then make sure we at least cut _something_, instead of leaving the surface solid`.

However, this seems like a VERY rare edge case, especially since `regularPolygonalHole` is only used by `fillHoles`, and now the dice box. That said, I don't really mind either way. For the purposes of the dice box, no one will ever set the radius to zero; I'm happy to set the radius however you wish. But perhaps that adjustment could be a separate commit, since related to the original implementation, instead of the addition of rounded corners.

The corner radius, on the other hand, could absolutely be zero, with no issue. At the very least, the `+ 1E-9` should be removed from the corner radius burn adjustment. I'll make this change now.